mcc/Dispatcher.c

```diff
@@ -159,6 +164,7 @@ static IPTR mNew(struct IClass *cl, Object *obj, struct opSet *msg)
             setFlag(data->flags, FLG_ActiveOnClick);
             setFlag(data->flags, FLG_PasteStyles);
             setFlag(data->flags, FLG_PasteColors);
+            setFlag(data->flags, FLG_HScroll);
```

Comment:
Is this really correct that you enable horizontal scrolling on per default? I think this should be dependent on the fact if a developer specified a horizontal slider upon object creation.